example6.py

```diff
@@ -0,0 +1,20 @@
+'''Задача 6: Вы пользуетесь общественным транспортом? 
+Вероятно, вы расплачивались за проезд и получали билет с номером. 
+Счастливым билетом называют такой билет с шестизначным номером, где сумма первых трех цифр равна сумме последних трех. 
+Т.е. билет с номером 385916 – счастливый, т.к. 3+8+5=9+1+6. Вам требуется написать программу, которая проверяет счастливость билета.
+
+*Пример:*
+
+385916 -> yes
+123456 -> no'''
+
+x = list(map(int, list(input())))
+
+left = sum(x[0:-3]) #с помощью среза получаем сумму от 0 элемента до 3 с конца
+right = sum(x[-3:]) #с помощью среза получаем сумму последних 3 чисел
```

Comment:
Самое забавное, что так будет тоже верно:
```suggestion
left = sum(x[:3]) #с помощью среза получаем сумму от 0 элемента до 3 с конца
right = sum(x[3:]) #с помощью среза получаем сумму последних 3 чисел
```